beacon_node/genesis/src/eth1_genesis_service.rs

```diff
@@ -183,7 +183,9 @@ impl Eth1GenesisService {
                 info!(
                     log,
                     "Genesis ceremony complete";
-                    "genesis_validators" => genesis_state.get_active_validator_indices(E::genesis_epoch()).len(),
+                    "genesis_validators" => genesis_state.get_active_validator_indices(E::genesis_epoch(), &spec)
+                    .map_err(|_e| String::from("EpochOutOfBounds"))?
+                    .len(),
```

Comment:
Rustfmt is a bit annoying in that it doesn't work under the logging macros, would you mind manually formatting the length calculation or moving it to its own line? The `map_err` could also be written more cleanly as `map_err(|e| format!("Genesis validators error: {:?}", e))?` (i.e. don't throw the error away).

Reply:
done!